### PR TITLE
courseware models always imports with lms.djangoapps prefix

### PR DESCRIPTION
**Pull request by bradmerlin**
As part of [this PR](https://github.com/open-craft/xblock-poll/pull/35), correctly import the Courseware app's models.

**JIRA tickets**: [OC-2981](https://tasks.opencraft.com/browse/OC-2981) [MCKIN-5433](https://edx-wiki.atlassian.net/browse/MCKIN-5433)

**Testing instructions**:

None

**Reviewers**
- [ ] @bradenmacdonald

### CODE REVIEW

**Comment by edx-webhook** [comment]
Thanks for the pull request, @bradmerlin! It looks like you're a member of a company that does contract work for edX. If you're doing this work as part of a paid contract with edX, you should talk to edX about who will review this pull request. If this work is not part of a paid contract with edX, then you should ensure that there is an OSPR issue to track this work in [JIRA](https://openedx.atlassian.net), so that we don't lose track of your pull request.

<a   href="https://openedx-webhooks.herokuapp.com/github/process_pr?repo=edx-solutions%2Fedx-platform&number=891"   rel="nofollow" >Create an OSPR issue for this pull request</a>.

File: lms/djangoapps/instructor/enrollment.py [hunk 1]
```diff
@@ -28,7 +28,7 @@
 from xmodule.modulestore.exceptions import ItemNotFoundError
 
 from course_modes.models import CourseMode
-from courseware.models import StudentModule
+from lms.djangoapps.courseware.models import StudentModule
```
Comment (bradenmacdonald) [inline lms/djangoapps/instructor/enrollment.py]: @bradmerlin  This file has some duplicate imports. I know they're not your fault, but it's causing the build error now that you've touched those lines. Can you please fix?

**Comment by bradenmacdonald** [comment]
@bradmerlin What was the reason for closing this?

**Comment by bradmerlin** [comment]
@bradenmacdonald please see https://tasks.opencraft.com/browse/OC-2981?focusedCommentId=47228&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-47228

**Pull request closed without merging**